django_loci/static/django-loci/js/loci.js

```diff
@@ -508,6 +518,14 @@ django.jQuery(function ($) {
         }
     });
 
+    var pathname = window.location.pathname;
+    if (pathname.includes('django_loci') || pathname.includes('testdeviceapp')) {
+        $coordsUrl = getCoordsUrl('django_loci');
+        $addrUrl = getAddrUrl('django_loci');
+    } else if (pathname.includes('geo') || pathname.includes('device')) {
+        $coordsUrl = getCoordsUrl('geo');
+        $addrUrl = getAddrUrl('geo');
+    }
```

Comment:
Unfortunately this is not a good approach, because "geo" is from openwisp-controller, you're adding logic that knows details of openwisp-controller, violating the loose coupling principle.
It would also mean that any other app which extends this library and is not called "geo" will not work.

This issue was already addressed for the other URLs to get the floorplan data, there's an inline template override which generates the URL using the django view name, these are retrieved with javascript.
See:
- https://github.com/openwisp/django-loci/blob/master/django_loci/templates/admin/django_loci/location_inline.html
- https://github.com/openwisp/django-loci/blob/master/django_loci/static/django-loci/js/loci.js#L34-L35

You simply have to be consistent with what's already done, please keep this in mind for all subsequent pull requests.